docker/configure_workers_and_start.py

```diff
@@ -770,7 +794,15 @@ def generate_worker_files(
 
     # A list of internal endpoints to healthcheck, starting with the main process
     # which exists even if no workers do.
-    healthcheck_urls = ["http://localhost:8080/health"]
+    # This list ends up being part of the command line to curl, which added unix socket
+    # support in version 7.40. The scheme and hostname are ignored, the path is not.
+    if using_unix_sockets:
+        healthcheck_urls = [
+            f"--unix-socket {MAIN_PROCESS_UNIX_SOCKET_PUBLIC_PATH} "
+            "http://localhost/health"
```

Comment:
Since the scheme/hostname are ignored, is there any standard way people usually define these (is `http://localhost` pretty canonical)?

Reply:
Not really, but the curl reference showed similar to this and it matched what was(essentially) already there. I believe it is canonical?

I think [this](https://curl.se/libcurl/c/CURLOPT_UNIX_SOCKET_PATH.html) was the page I used for that info.